File: examples-sdk/java/201-llm-basic-chat/Main.java
// 201-llm-basic-chat — Java SDK equivalent
// Compile: vil compile --from java --input 201-llm-basic-chat/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("llm-basic-chat", 3100);
        ServiceProcess chat = new ServiceProcess("chat");
        chat.endpoint("POST", "/chat", "chat_handler");
        server.service(chat);
        server.compile();
    }
}

File: examples-sdk/kotlin/201-llm-basic-chat/main.kt
// 201-llm-basic-chat — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 201-llm-basic-chat/main.kt --release

fun main() {
    val server = VilServer("llm-basic-chat", 3100)
    val chat = ServiceProcess("chat")
    chat.endpoint("POST", "/chat", "chat_handler")
    server.service(chat)
    server.compile()
}
